Jenkinsfile

```diff
@@ -16,6 +16,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 //@Library(value="pipeline-lib@my_branch_name") _
+@Library(value="pipeline-lib@marosale/CORCI-951-refactor") _
```

Comment:
I don't think you changed anything that would requires this line.

Reply:
Thanks for your feedback. I removed this line